ci/LDKSwift/Package.swift

```diff
@@ -4,66 +4,65 @@
 import PackageDescription
 import Foundation
 
+var cBindingsBase = "/Users/arik/Developer/ldk-c-bindings"
```

Comment:
Could this be something like `../../ldk-c-bindings` assuming that most people would clone the repos at the same level? Not sure if that would work...

Reply:
I'm not sure exactly how to address this yet due to some additional upcoming changes that are gonna be necessary for the framework generation